src/mobile/apps/show/helpers/view_helpers.coffee

```diff
@@ -0,0 +1,45 @@
+_ = require 'underscore'
+{ slugify } = require 'underscore.string'
+{ compactObject } = require '../../../models/mixins/compact_object.coffee'
+sd = require('sharify').data
+
+module.exports =
+
+  toJSONLD: (show) ->
+    compactObject {
+      "@context": "http://schema.org"
+      "@type": "Event"
+      name: show.get('name')
+      image: show.get('image_url')
+      url: "#{sd.APP_URL}/show/#{show.get('id')}"
+      startDate: (new Date(show.get('start_at'))).toISOString()
+      endDate: (new Date(show.get('end_at'))).toISOString()
+      location: @toJSONLDLocation(show.location()) if show.location()
```

Comment:
If there's no location we should make sure the entire event doesn't output as it's a required field.

Reply:
actually that was being verified [here](https://github.com/artsy/force/pull/4892/commits/97b7486827897dea563a7ea23af4c7639e3ef154#diff-878c5a73df4d8c8b6e7dfbdaf6a40188R43) but moved it to the helper to make it more clear. Also added [test](https://github.com/artsy/force/pull/4892/commits/c73f1549d4ec1cbba2ed79dce58e81f3c669b56e#diff-58f0ca1d531f85ee57caed74e0aea66fR30) for that.